promotion/datastore_test.go

```diff
@@ -257,14 +257,18 @@ func (suite *PostgresTestSuite) TestGetAvailablePromotionsForWallet() {
 	suite.Assert().True(promotions[0].Available)
 	suite.Assert().False(promotions[1].Available)
 
-	_, err = pg.CreateClaim(promotion.ID, w.ID, decimal.NewFromFloat(30.0), decimal.NewFromFloat(0))
+	adClaimValue := decimal.NewFromFloat(30.0)
+	adSuggestionsPerGrant := 30 * 4
```

Comment:
does it make sense for this to be divided by the claim value by the suggestions per grant so we don't have to change it in the future if we have to change the # of suggestions per grant?